BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs

```diff
@@ -8,11 +16,38 @@ namespace BackendApartmentReservation.Controllers
     [ApiController]
     public class ProfilesController : ControllerBase
     {
+        private readonly UserManager<DbEmployee> _userManager;
         [HttpGet]
         [Route("profiles")]
         public async Task<IEnumerable<string>> Get()
         {
             return await Task.FromResult(new[] { "userProfile1", "userProfile2" });
         }
+
+        [HttpPost]
+        [AllowAnonymous]
+        [Route("register")]
```

Comment:
According to REST API conventions, verbs should be avoided in routes. In my opinion, it is better to leave simply `POST /profiles`. Discussion on that is welcome :)